addons/packages/metrics-server/0.5.1/package-template.yaml

```diff
@@ -0,0 +1,30 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
```

Comment:
@blc1996 , I was asked to remove the package-template.yaml file from my PR, so we can follow the same trend here

Reply:
removed